src/NJsonSchema.Annotations/JsonFormatStrings.cs

```diff
@@ -0,0 +1,84 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="JsonFormatStrings.cs" company="NJsonSchema">
+//     Copyright (c) Rico Suter. All rights reserved.
+// </copyright>
+// <license>https://github.com/RicoSuter/NJsonSchema/blob/master/LICENSE.md</license>
+// <author>Rico Suter, mail@rsuter.com</author>
+//-----------------------------------------------------------------------
+
+using System;
+
+namespace NJsonSchema.Annotations;
+
+/// <summary>Class containing the constants available as format string. </summary>
+public static class JsonFormatStrings
```

Comment:
I wonder whether we should put this in the NJsonSchema namespace but the NJsonSchema.Annotations pkg
and remove from NJsonSchema (and the same with JsonObjectType). What do you think?

Reply:
I understand the motive, but it feels strange to exclude one attribute from the annotations package.

The duplicity is there because of the Newtonsoft attribute. I can solve it instead with JsonConverter for that type. What do you think?

Reply:
Sorry no my propsal: 

JsonFormatStrings: Keep in NJsonSchema.Annotations but with namespace NJsonSchema (remove in NJsonSchema pkg)
JsonObjectType: Keep in NJsonSchema.Annotations but with namespace NJsonSchema (remove in NJsonSchema pkg)

Reply:
That's ok for me. How do you prefer to handle these [attributes](https://github.com/RicoSuter/NJsonSchema/pull/1641/files#diff-a21a5e443cda6224b4a0f1391771bfad4d1fd8f3b146a01879fa7fcf4a23e9fcL35)?

Reply:
They are from Newtonsoft package

Reply:
These? Link seems to be broken...

Reply:
These: `JsonPropertyAttribute` 

https://github.com/RicoSuter/NJsonSchema/blob/8b30fcd633389a56c10ac23c9287348775f849a8/src/NJsonSchema/JsonObjectType.cs#L26C1-L28C19

```cs
using Newtonsoft.Json;

namespace NJsonSchema
{
    /// <summary>
    /// Enumeration of the possible object types.
    ///
    /// Keep in sync with <see cref="Annotations.JsonObjectType"/>
    /// </summary>
    [Flags]
    public enum JsonObjectType
    {
        /// <summary>No object type. </summary>
        [JsonProperty("none")] // <----------------------------- this ---------------<
        None = 0,

        /// <summary>An array. </summary>
        [JsonProperty("array")] // <----------------------------- and this ---------------<
        Array = 1,
...
    }
}
```

Reply:
seems that these are not needed because of the magic here: 

![image](https://github.com/RicoSuter/NJsonSchema/assets/2603405/881bd3d7-9fea-49cb-ade8-cef13b7a236d)

Reply:
Took me some time to find it :)